### DISCUSSION TITLE
Firebase key error - how to resolve this

### DISCUSSION THREAD

**Discussion by PiyushKalyanpy** [PiyushKalyanpy/GyanaGuru#54] [General]
Contributors are getting this error. This error can be resolved by adding the api key's. but for the contributors is there any easy options to remove this error or anything we can do so that only those contributor will see this those who are contributing in backend part.

![image](https://github.com/PiyushKalyanpy/GyanaGuru/assets/79275157/201ece51-bb3c-4fc3-b17b-293876c6ee0b)

**Comment by ujjawaltyagii** [comment]
@PiyushKalyanpy can we add the database\firebase.js to `.gitignore` so that when we push code so we dont have to worry about API key. What do you think ?

**Comment by afeefuddin** [comment]
how do i solve this error btw unable to run it locally

**Reply by ujjawaltyagii** [reply]
you need to create the project in firebase and then from there you need to paste firebase api key then it will work
here is reference :

![guide](https://github.com/PiyushKalyanpy/GyanaGuru/assets/115401171/a0163f7c-538c-4e02-88f5-982d79a83fff)
